### ISSUE TITLE
ObservableCollection does not work correctly with the library

### ISSUE THREAD

**Issue by MajorXAML** [lepoco/wpfui#63]
If we try to add any control via code using ObservableCollection, we won't get the desired result.
for example:
```
public ObservableCollection<WPFUI.Controls.Card> TestColl { get; set; } = new ObservableCollection<WPFUI.Controls.Card>();

TestTest.Add(new WPFUI.Controls.Card {Content = "Test"});
```
The result will be just Test text without any style

Similar behavior can be seen in the Demo > Controls > DataGrid
![data grid](https://user-images.githubusercontent.com/93913816/154148348-c0f7c974-25fd-41d5-ac4b-d21e45d20c85.PNG)
As you see the checkboxes use the default windows style.

Maybe someone knows why this happens? I have not yet tested other ui libraries, wonder how they handle it.

**Comment by MajorXAML** [comment]
Quick update.
I checked ModernWPF and adding controls via code works without any problems! I have an assumption that the problem is in the way we connect the library. Modernwpf does this using xmlns 🤨

**Comment by pomianowski** [comment]
At the moment, the DataGrid is simply added with the background and basic stuff, I did not go into this control because it requires a thorough redesign. In free time, it will be rewritten

**Comment by MajorXAML** [comment]
> At the moment, the DataGrid is simply added with the background and basic stuff, I did not go into this control because it requires a thorough redesign. In free time, it will be rewritten
Yes, I understand. But I pointed out the CheckBoxes. Any control we add through code will look completely unstyled.

**Comment by pomianowski** [comment]
Binding controls seems to just work, DataGrid has own styles for CheckBoxes etc
![image](https://user-images.githubusercontent.com/13592821/154319069-e4c4a53f-6f9e-44b9-b1d7-95267278b02c.png)

**Comment by MajorXAML** [comment]
![CollTest](https://user-images.githubusercontent.com/93913816/154368490-b784f556-d2cc-4195-9c92-298f7f8de853.PNG)
I think I figured it out, this happens only with ItemsControl, for some unknown reason ItemContainerStyle resets all styles 
of all the controls inside... This didn't happen in modernwpf or when using UserControls. But I think I can find a workaround for my case. 

As for DataGrid, it's a similar situation. DataGrid should use global styles for controls (like checkbox) provided by the library. But it feels like it's just isolated from other UI elements... I checked DataGrid in modernwpf and the checkboxes are fine. I can replace them with any other control and it will work.
![datagrid](https://user-images.githubusercontent.com/93913816/154368784-155e45dd-f01c-4a82-a0bd-a2783fa4d042.PNG)

Anyway, thanks for helping me figure out the cause! If I find a way to fix datagrid I will let you know.
